### PR TITLE
Add CodeRadio support

### PR DESCRIPTION
**Pull request by papposilene**
Radio URL: https://coderadio.freecodecamp.org/
Description: 24/7 music designed for coding.

**Describe the changes you made**
Add connector for CodeRadio.

**Additional context**
One typo correction.

### CODE REVIEW

File: src/connectors/coderadio.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+'use strict';
+
+Connector.playerSelector = '.now-playing';
+Connector.artistSelector = '.now-playing div[data-meta="artist"]';
+Connector.trackSelector = '.now-playing div[data-meta="title"]';
+Connector.albumSelector = '.now-playing div[data-meta="album"]';
+
+Connector.isPlaying = () => document.getElementById('toggle-play-pause').getAttribute('aria-label') !== 'Play';
+
+Connector.onReady = Connector.onStateChanged;
```
Comment (inverse) [inline src/connectors/coderadio.js:13]: I don't think this line is needed

File: src/core/content/util.js [hunk 2]
```diff
@@ -327,7 +327,7 @@ const Util = {
 	 * a first element with the attribute available.
 	 *
 	 * @param  {String|Array} selectors Single selector or array of selectors
-	 * @param  {String} attr Attrubute to get
+	 * @param  {String} attr Attribute to get
```
Comment (inverse) [inline src/core/content/util.js:330]: Ideally these other changes should go into seperate pull requests so we can track changes independently.

Comment (papposilene) [reply src/core/content/util.js]: I will push a new merge for this typo :)

**Review by papposilene — COMMENTED** [review]
All done.

**Review by inverse — APPROVED** [review]

**Comment by inverse** [comment]
Thanks 🙏🏼

**Pull request merged**